test/sys/test_termios.rs

```diff
@@ -19,3 +24,51 @@ fn test_tcgetattr() {
         }
     }
 }
+
+#[test]
+fn test_cfmakeraw() {
+    let mut termios: termios::Termios = unsafe { mem::uninitialized() };
+    termios::cfmakeraw(&mut termios);
+}
+
+// Test ergonomics of using character indices
+#[test]
+fn test_char_indices() {
+    use nix::sys::termios::SpecialCharacterIndices::VEOF;
+    use nix::sys::termios::_POSIX_VDISABLE;
+
+    let master_fd = posix_openpt(O_RDWR).unwrap();
+    assert!(master_fd.as_raw_fd() > 0);
+    let mut termios = termios::tcgetattr(master_fd.as_raw_fd()).unwrap();
+    termios.c_cc[VEOF as usize] = _POSIX_VDISABLE;
```

Comment:
It doesn't provide test coverage. As the comment at the start of the function describes, it's merely there to check the ergonomics. I wanted to make sure my API easily supported modifying the `termios.c_cc` array using the enums and constants I defined. I wanted to make sure everything for termios is imported from `libc` and uses safe datatypes that still allow for all normal usage. It might make more sense to move this (and the other "test" I implemented) to a doc comment instead of a separate test.